packages/graphql-transformers-e2e-tests/src/__tests__/AuthV2ExhaustiveT3A.test.ts

```diff
@@ -0,0 +1,88 @@
+import { AuthProvider, AuthStrategy, ModelOperation } from '@aws-amplify/graphql-auth-transformer';
```

Comment:
`AuthV2ExhaustiveT3A.test.ts` are pretty much copy of `AuthV2ExhaustiveT2A.test.ts` but with new setting.

This is to assert that iam directive works as before while new setting is one.

(this applies to 3 similar tests)